contrib/win32/openssh/config.h.vs

```diff
@@ -1768,5 +1768,3 @@
 #define HAVE_EVP_PKEY_GET0_RSA 1
 #define HAVE_EVP_MD_CTX_NEW 1
 #define HAVE_EVP_MD_CTX_FREE 1
-
```

Comment:
Why are we seeing these changes?

Reply:
I noticed a few extra lines at the end of the file while I was re-doing the merge, so I removed them